### ISSUE TITLE
Jj

### ISSUE THREAD

**Issue by ko973** [mattbierner/urban-dictionary-word-list#8]
base-uncased-4096") + +SENTENCE = "This is a test for sequence classification. " * 300 +token_ids = tokenizer( + SENTENCE, + return_tensors="pt", + padding="max_length", # Optional but recommended + truncation=True # Optional but recommended + ) +output = model(**token_ids) + +> SequenceClassifierOutput(loss=None, logits=tensor([[-0.3051, -0.1762]], grad_fn=), hidden_states=None, attentions=None) + + +## Training global tokens +To train global tokens and the classification head only: +python: +from transformers import AutoModelForSequenceClassification, AutoTokenizer + +model = AutoModelForSequenceClassification.from_pretrained("ccdv/legal-lsg-base-uncased-4096", + trust_remote_code=True, + pool_with_global=True, #
